### PR TITLE
Improve Unit Test coverage

### PR DESCRIPTION
**Pull request by ryanlerch**
100% test coverage for server/views/generic.py 
100% test coverage for server/util.py 
100% test coverage for server/views/validators.py

### CODE REVIEW

**Comment by ryanlerch** [comment]
Throwing the WIP label on this one, as i am going to try to 100% another one :)

File: bodhi/server/validators.py [hunk 1]
```diff
@@ -454,7 +454,7 @@ def validate_acls(request, **kwargs):
             release = cache_release(request, build)
             if release is None:
                 return
-        elif 'update' in request.validated:
+        else:
```
Comment (cverna) [inline bodhi/server/validators.py]: Is this change needed ? I would be cautious about changing the logic in this validator :-)

Comment (ryanlerch) [reply bodhi/server/validators.py]: with the nature of the checker, there never seems to be a case where we would be able to get this far, and not have update in request.validated. I'll pull this change out though, but validators.py won't be 100% covered anymore...

File: bodhi/server/util.py [hunk 2]
```diff
@@ -119,19 +119,6 @@ def build_evr(build):
     return tuple(map(str, (build['epoch'], build['version'], build['release'])))
 
 
-def link(href, text):
```
Comment (Zlopez) [inline bodhi/server/util.py:122]: Why is this removed? It is not called anymore?

**Review by Zlopez — COMMENTED** [review]
Just one comment, otherwise looks good.

File: bodhi/server/util.py (hunk 2 above)
Comment (ryanlerch) [reply bodhi/server/util.py:122]: yeah, this one is unused now.

**Review by Zlopez — APPROVED** [review]
